Frontend/ui-driver/src/Strings/BN.js

```diff
@@ -247,6 +247,11 @@ const bengaliStrings = {
 	UPDATED_AT: "ম্যাপ শেষ আপডেট",
 	TRIP_COUNT: "আজকের ট্রিপস",
 	TODAYS_EARNINGS: "আজকের উপার্জন",
+	BONUS_EARNED : "বোনাস অর্জিত",
+	GOT_IT : "বুঝেছি!",
+	WHAT_IS_NAMMA_YATRI_BONUS : "নম্মা যাত্রী বোনাস কি?",
+	BONUS_PRIMARY_TEXT : "নম্মা যাত্রী বোনাস হল অতিরিক্ত পরিমাণ যা আপনি মিটার চার্জের উপরে পিকআপ চার্জ, গ্রাহকের টিপস এবং ড্রাইভার যোগের আকারে অর্জন করেছেন।",
+    BONUS_SECONDARY_TEXT : "নম্মা যাত্রী বোনাসের পরিমাণ আপনার মোট উপার্জনের অংশ।",
```

Comment:
@aryanguptaSG please try to maintain indentation.

Reply:
it is already indented perfectly don't know why it is looking like this here .